woocommerce/woocommerce
# restore $attachment_ids for product thumbnails template

### All Submissions:


### Changes proposed in this Pull Request:

This is a followup PR to #27289. It updates the template versions to the correct version and fixes a warning created in the original PR. Ref https://github.com/woocommerce/woocommerce/commit/e49527cf9d546985a4f9c6db284d9540de302efb#r43055263

### How to test the changes in this Pull Request:

1. Create a product and assign multiple gallery images
2. View the single product
3. First gallery image should be used as product image and not be duplicated in the list of gallery images
4. Assign a product image
5. Product image & gallery should be displayed the same as in master
6. Image zoom should work in all testing
7. There should be no debug warnings logged.


### Other information:




### Changelog entry

Patches a fix for 4.7
